src/react/components/Provider.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ReactMediaChromeContext } from './Context.js';
+import createMediaStore from '../../js/mediaStore.js';
+const mediaUIStore = createMediaStore({})
+function Provider({
+  // mediaUIStore = createMediaStore({}),
+  context,
+  children,
+  // serverState,
+}) {
+  const Context = context || ReactMediaChromeContext;
+
+  // In a real version of this, we may want to make the store overridable and rely on some memoization "under the hood"
+  return React.createElement(
+    Context.Provider,
+    { value: mediaUIStore },
+    children
+  );
+}
+
+export default Provider;
```

Comment:
Call this `MediaContextProvider` instead?

Reply:
Since this is a default export, **_typically_** they are just exported as `Provider` and you can call it whatever you'd like. I believe that's the norm. Also, fwiw, all React `<Provider>`s are context providers (by definition). That said, happy to rename.